### ISSUE TITLE
The psnr value can‘t reach 32.84 on Set14 (SESR-M5, f=16)

### ISSUE THREAD

**Issue by HY-2837** [ARM-software/sesr#13]
Hello! I trained sesr (M5, f=16) on div2k, but when I tested on set14, the psnr value was 32.6, can't reach 32.84 (from paper table 1). I set the learning rate to 5e-4 (from paper) and 1e-4 (from github code), but none of them can achieve the effects mentioned in the paper table 1. Do I need change some hyperparameters？ Could you provide your trained "SESR-M5- f=16" model？
email：cuihezdh@gmail.com

**Comment by eladc-git** [comment]
Hi, I got the same issue. What are the hyperparameters that achieve psnr=32.84 for Set14 as in your paper?

**Comment by prathapsoma** [comment]
Can you  suggest me please how to test the network after training.
